includes/lib/class-faster-image-b52f1a8-faster-image.php

```diff
@@ -0,0 +1,189 @@
+<?php
+
+require_once( AMP__DIR__ . '/includes/lib/class-faster-image-b52f1a8-invalid-image-exception.php' );
+require_once( AMP__DIR__ . '/includes/lib/class-faster-image-b52f1a8-image-parser.php' );
+require_once( AMP__DIR__ . '/includes/lib/class-stream-17b32f3-stream.php' );
+require_once( AMP__DIR__ . '/includes/lib/class-stream-17b32f3-stream-buffer-too-small-exception.php' );
+
+/**
+ * FasterImage - Because sometimes you just want the size, and you want them in
+ * parallel!
+ *
+ * Based on the PHP stream implementation by Tom Moor (http://tommoor.com)
+ * which was based on the original Ruby Implementation by Steven Sykes
+ * (https://github.com/sdsykes/fastimage)
+ *
+ * MIT Licensed
+ *
+ * @version 0.01
+ */
+class Faster_Image_B52f1a8_Faster_Image
+{
+    /**
+     * The default timeout
+     *
+     * @var int
+     */
+    protected $timeout = 10;
```

Comment:
Agreed, I've changed it to 4 for starters. I was inclined to lower it right to 2. What do you think?